docs/topics/api/hero.rst

```diff
@@ -15,13 +15,15 @@ Combined Hero Shelves
 
 .. _hero-shelves:
 
-This convienence endpoint serves a single, randomly selected, primary hero shelf.
+This convienence endpoint serves a single, randomly selected, primary hero shelf,
```

Comment:
the comma at the end shouldn't be needed?

Reply:
Depends if you think there should be a pause in the sentence.